plugins/jj/jj.plugin.zsh

```diff
@@ -0,0 +1,55 @@
+# if jj is not found, don't do the rest of the script
+if (( ! $+commands[jj] )); then
+  return
+fi
+
+source <(jj util completion zsh)
+compdef _jj jj
+
+function __jj_prompt_jj() {
+  local flags=("--no-pager")
+  if (( ${ZSH_THEME_JJ_IGNORE_WORKING_COPY:-0} )); then
+    flags+=("--ignore-working-copy")
+  fi
```

Comment:
Could you refactor this into a `zstyle`-like config? You can find one in `yarn` plugin, for example.